app/deployment/app.go

```diff
@@ -307,10 +307,10 @@ func (a *app) doDeliverCreateTx(ctx apptypes.Context, tx *types.TxCreateDeployme
 			Resources:    group.Resources,
 			OrderTTL:     tx.OrderTTL,
 		}
-		a.State().DeploymentGroup().Save(g)
+		state.DeploymentGroup().Save(g)
```

Comment:
Error return value of `(github.com/ovrclk/akash/state.DeploymentGroupAdapter).Save` is not checked